include/deal.II/dofs/dof_tools.h

```diff
@@ -1198,29 +1198,6 @@ namespace DoFTools
    * @{
    */
 
-  /**
-   * Select all dofs that will be constrained by interface constraints, i.e.
-   * all hanging nodes.
-   *
-   * The size of @p selected_dofs shall equal <tt>dof_handler.n_dofs()</tt>.
-   * Previous contents of this array or overwritten.
-   *
-   * In case of a parallel::shared::Triangulation or a
-   * parallel::distributed::Triangulation only locally relevant dofs are
-   * considered. Note that the vector returned through the second argument still
-   * has size <tt>dof_handler.n_dofs()</tt>. Consequently, it can be very large
-   * for large parallel computations -- in fact, it may be too large to store on
-   * each processor. In that case, you may want to choose the variant of this
-   * function that returns an IndexSet object.
```

Comment:
I think we need to copy this part of the documentation to the function that remains (as that function only says "same as above") - I think the documentation is already there on the other few functions, but here it apparently got missed.

Reply:
Done.